### ISSUE TITLE
Error: cannot add bindings to a locked environment

### ISSUE THREAD

**Issue by ccruizm** [romanhaa/Cerebro#29]
Good day, 

Thanks for developing such a great tool. I have created my `.crb` file in our HPC and downloaded it to my laptop where I download the `Cerebro` app. However, when loading the dataset, I get an error. What can be the cause of this problem? I have the impression that this is a standalone app that can run independently (I do not have R installed on my local computer).

Thanks in advance for your help!

<img width="1193" alt="Screenshot 2020-12-16 at 17 44 04" src="https://user-images.githubusercontent.com/37718031/102378620-4e9e3e00-3fc6-11eb-9c68-bd4ba2849f8b.png">

**Comment by ccruizm** [comment]
Sorry, after I careful reading of other issues and the web app, my problem is likely due to the version I used to generate my `crb` file (v1.3). it seems I have to be patient until the new app is released! looking forward to it

**Comment by romanhaa** [comment]
Can you install cerebroApp (the R package) on your local machine? I think that would be the easiest solution. You could then generate the `.crb` file with v1.3 on the HPC, transfer it to your local machine and load it into Cerebro v1.3 that you launch from within R using [this](https://romanhaa.github.io/cerebroApp/reference/launchCerebro.html) function.

**Comment by Sirt0r** [comment]
I also have the same problem. For myself I would like to use Cerebro to help others from my team to interact with these types of analyses on our own server. So If I try to load the .crb file in my R session, it works perfectly fine. However it does not on the server with installed via docker. Do you think you could look at your docker container again?

**Comment by romanhaa** [comment]
Right, there is no Docker container with cerebroApp v.1.3 installed in it. The most recent is based on v1.2.2. I will try to get around creating one this weekend.

**Comment by romanhaa** [comment]
Quick update: There is now a Docker image running Cerebro v1.3.0. You can find it [here](https://hub.docker.com/repository/docker/romanhaa/cerebro) with tag `v1.3.0_light` (light because it only contains cerebroApp and not Seurat, so you can't use it for analysis). The `latest` tag refers to the same image. You can launch a container like this: `docker run -it -p 8080:8080 --rm romanhaa/cerebro:v1.3.0`

Let me know if that helps.

**Comment by Sirt0r** [comment]
Perfect. It runs smoothly now and I can view the files through the server. Thank you very much.

**Comment by ccruizm** [comment]
I have tried to use the docker image but our server does not have installed Docker for security reasons and we use singularity. I could not find how to do port forwarding using singularity. Do you have maybe any ideas on how I could do this, please? btw, any news on the portable app? that would be a huge help (and easier to share for people with no bioinformatics background)!

Thanks!
